package.json

```diff
@@ -5,14 +5,19 @@
   "main": "index.ts",
   "scripts": {
     "build": "tsc -p .",
-    "start": "ts-node index.ts"
+    "start": "ts-node src/index.ts",
+    "dev": "nodemon --exec ts-node src/index.ts",
+    "migrate:make": "knex --knexfile src/database/knexfile.ts -x ts migrate:make",
+    "migrate:latest": "knex --knexfile src/database/knexfile.ts migrate:latest",
+    "seed:make": "knex --knexfile src/database/knexfile.ts -x ts seed:make",
```

Comment:
I would totally love a db reset script which does migrate and seeds, dropdb and re create it